tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -492,36 +493,41 @@ void test_sphere() {
 
   const std::array<
       std::variant<creators::Sphere::Excision, creators::Sphere::InnerCube>, 3>
-      interiors{{creators::Sphere::Excision{}, creators::Sphere::InnerCube{0.0},
-                 creators::Sphere::InnerCube{0.7}}};
+      interiors{{creators::Sphere::InnerCube{0.0},
+                 creators::Sphere::InnerCube{0.7},
+                 creators::Sphere::Excision{}}};
   const std::array<
       std::optional<creators::Sphere::EquatorialCompressionOptions>, 2>
       equatorial_compressions{
           {std::nullopt,
            creators::Sphere::EquatorialCompressionOptions{0.5, 2}}};
-  const std::array<std::vector<double>, 2> radial_partitioning{
-      {{}, {0.5 * (inner_radius + outer_radius)}}};
-  const std::array<std::vector<domain::CoordinateMaps::Distribution>, 2>
+  const double outer_minus_inner = outer_radius - inner_radius;
+  const std::array<std::vector<double>, 3> radial_partitioning{
+      {{},
+       {0.5 * (inner_radius + outer_radius)},
+       {inner_radius + 0.333 * outer_minus_inner,
+        inner_radius + 0.666 * outer_minus_inner}}};
+  const std::array<std::vector<domain::CoordinateMaps::Distribution>, 3>
       radial_distribution{
           {{domain::CoordinateMaps::Distribution::Linear},
            {domain::CoordinateMaps::Distribution::Linear,
-            domain::CoordinateMaps::Distribution::Logarithmic}}};
+            domain::CoordinateMaps::Distribution::Logarithmic},
+           {domain::CoordinateMaps::Distribution::Logarithmic}}};
 
   const std::array<double, 3> velocity{{2.3, -0.3, 0.5}};
   const std::vector<double> times{1., 10.};
 
-  for (const auto& [interior_index, equiangular, equatorial_compression,
-                    array_index, which_wedges, time_dependent,
-                    with_boundary_conditions] :
+  for (const auto& [equiangular, equatorial_compression, array_index,
+                    which_wedges, time_dependent, with_boundary_conditions] :
        random_sample<5>(
            cartesian_product(
-               make_array(0_st, 1_st, 2_st), make_array(false, true),
-               equatorial_compressions, make_array(0.0, 1.0),
+               make_array(false, true), equatorial_compressions,
+               make_array(0_st, 1_st, 2_st),
                make_array(ShellWedges::All, ShellWedges::FourOnEquator,
                           ShellWedges::OneAlongMinusX),
                make_array(true, false), make_array(true, false)),
            make_not_null(&gen))) {
-    const auto& interior = interiors[interior_index];
```

Comment:
Don't couple the interior selection to the radial distribution selection. I don't know why `array_index` was a double here, but just make it three size_ts.